paper/paper.tex

```diff
@@ -169,6 +169,10 @@ \subsection{Main features of Trixi.jl}
   \end{itemize}
 \end{itemize}
 
+Currently, \trixi provides shared-memory parallelization via multithreading.
+Distributed-memory parallelization with MPI is available for some mesh types
+but still work in progress.
```

Comment:
```suggestion
Distributed-memory parallelization with MPI is available for some mesh types;
increasing the capability of this feature is subject to ongoing work.
```

I am not 100% happy with this rewording but the original sounded awkward.

Reply:
Maybe something in this direction?
```suggestion
Initial parallelization with MPI is available for some mesh types,
but full support for distributed-memory parallelism is subject to ongoing work.
```
But what @andrewwinters5000 wrote sounds good to me too.